docs/pages/identity-governance/entra-id.mdx

```diff
@@ -0,0 +1,359 @@
+---
+title: Entra ID Integration
+description: Describes how to set up the Teleport Entra ID integration in Teleport.
+labels:
+ - how-to
+ - identity-governance
+---
+
+Entra ID integration enables the following features in Teleport:
+
+1. **Single sign-on (SSO):** Configures Teleport authentication with Entra ID as an
+   identity provider.
+2. **User sync:** Imports Entra ID users as Teleport users.
+3. **Group sync:** Imports Entra ID groups as Teleport
+   Access Lists. 
+4. **Application sync (Optional):** Imports Entra ID applications. 
+5. **Integration with Teleport Identity Security (Optional):** Lets you analyze user access paths
+and policies from the Teleport Identity Security product. 
+
+## How it works
+
+An enterprise application will be configured in the Entra ID. 
+The purpose of this application is twofold - 
+1. Set up a SAML SSO provider for Teleport. 
+2. Set up Microsoft graph API credential for Teleport that allows to fetch resources 
```

Comment:
This is not accurate if using system credentials and we should enphasize that here to avoid confusion. 

The second case is only true if using OIDC for authentication, otherwise that's not required at all.